cv32e40p_manifest.flist

```diff
@@ -24,7 +24,9 @@
 ///////////////////////////////////////////////////////////////////////////////
 
 +incdir+${DESIGN_RTL_DIR}/include
++incdir+${DESIGN_RTL_DIR}/../bhv
 +incdir+${DESIGN_RTL_DIR}/../bhv/include
++incdir+${DESIGN_RTL_DIR}/../sva
```

Comment:
Please make sure that the assertions are included in the Makefile from example_tb/core as well

Reply:
Sorry, here I will have to say "no".  I am not the owner/maintainer of example_tb and in fact have asked several times for it to be removed.